Injecting Project Fails Nuget Build

## Background / Context *(Optional)*
When having a project with shared code is referenced by a packable project, the build Nuget package fails

## Reproduction Steps *(Required)*
- Add project reference to packable project and compile references
- see the fail in the workflow action run

## Expected Behaviour *(Required)*
- The Nuget package is built without the shared project reference

## Error Logs / Stack Trace *(Optional)*
Any relevant logs or stack traces that may help diagnose the bug.

```
Error: /usr/share/dotnet/sdk/10.0.201/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.targets(262,5): error NETSDK1085: The 'NoBuild' property was set to true but the 'Build' target was invoked. [/home/runner/work/postgres-library/postgres-library/src/CrowtherHoldings.Extensions.Postgres.Core/CrowtherHoldings.Extensions.Postgres.Core.csproj]
Error: Process completed with exit code 1.
```

--- a/actions/build-push-nuget/action.yml
+++ b/actions/build-push-nuget/action.yml
@@ -36,7 +36,6 @@ runs:
         set -e
         dotnet pack ${{ inputs.packagesToPack }} \
           --configuration ${{ inputs.buildConfiguration }} \
-          --no-build \
           --output "${{ github.workspace }}/nupkgs" \
           /p:PackageVersion="${PACKAGE_VERSION}" \
           --verbosity minimal

